packages/sapience/src/components/forecasting/forms/inputs/WagerInput.tsx

```diff
@@ -39,23 +77,12 @@ export function WagerInput({
     trigger,
     setValue,
   } = useFormContext();
+  const { chainShortName, marketGroupData, chainId } = useMarketGroupPage();
 
-  // Determine helper text based on collateral symbol
-  const helperText =
-    collateralSymbol === 'sUSDS' ? (
-      <>
-        sUSDS is the yield-bearing token of the Sky Protocol, currently earning
-        4.5% APY.{' '}
-        <a
-          href="https://swap.defillama.com/?chain=base&from=_&tab=swap&to=0x5875eee11cf8398102fdad704c9e96607675467a"
-          target="_blank"
-          rel="noopener noreferrer"
-          className="text-primary underline"
-        >
-          Get sUSDS
-        </a>
-      </>
-    ) : undefined;
+  const collateralSymbol = marketGroupData?.collateralSymbol || '';
+  const collateralAddress = marketGroupData?.collateralAsset as
+    | `0x${string}`
+    | undefined;
```

Comment:
instead of using the useMarketGroupPage, pass this data in.  This way wecan re-use WagerInput isn't context dependant (for when we move the component into the `ui` directory for example)